packages/xy-chart/src/utils/chartUtils.js

```diff
@@ -70,11 +70,11 @@ export function collectDataFromChildSeries(children) {
   Children.forEach(children, (Child, i) => {
     if (Child && Child.props && Child.props.data) {
       const name = componentName(Child);
-      let { data } = Child.props;
+      const { data } = Child.props;
       if (data && isSeries(name)) {
-        if (isCirclePackSeries(name)) {
-          data = computeCirclePack(data);
-        }
+        // if (isCirclePackSeries(name)) {
+        //   data = computeCirclePack(data);
```

Comment:
I think if we were going to modify the `yScale` it'd be best to do it here, then there is only one `yScale`. passing around 2 (one to `HorizontalRefLine` and one to `CirclePackSeries`) seems a little messy.

Reply:
pointed to a wrong place.